dns-global-exclusive-tls.ks.in

```diff
@@ -0,0 +1,81 @@
+# Test the e2e functionality of global exclusive encrypted DNS functionality
+# Partial tests of reqiured features are in 'certificate' and 'dns-global-bootopts'
+# tests.
+
+# Use defaults, but no network
+%ksappend common/common.ks
+%ksappend repos/default.ks
+%ksappend l10n/default.ks
+%ksappend users/default.ks
+%ksappend storage/default.ks
+%ksappend payload/default_packages.ks
+
+%certificate --filename tls-ca-bundle.pem --dir /etc/pki/dns/extracted/pem
+%ksappend data/edns-idmops.psi.redhat.com.crt
+%end
+
+# Reboot the installed system.
+reboot
+
+# Validate on the first boot.
+%ksappend validation/success_on_first_boot.ks
+
+
+%post --nochroot
```

Comment:
Shouldn't be the results logged to `/mnt/sysroot/root/RESULT` when running in nochroot?

Reply:
Updated